jolpica_api/ergastapi/tests/test_views.py

```diff
@@ -69,6 +69,28 @@ def test_viewsets(client: APIClient, endpoint: str, path: Path, django_assert_ma
 
     assert result == expected
 
+@pytest.mark.parametrize(
+    ["endpoint1", "endpoint2"],
+    [
+        # Check json & non-json formats return same data
+        ("circuits/monza", "circuits/monza.json"),
+        ("drivers/alonso/status/1", "drivers/alonso/status/1.json"),
+        ("2017/results/2?limit=2", "2017/results/2.json?limit=2"),
```

Comment:
I've moved these tests into their own test case.
Your changes were much more thorough, however they increased testing time by ~25%, which I'd like to avoid if possible. Hope that makes sense